src/pages/feedback/FeedbackPage.tsx

```diff
@@ -1,60 +1,63 @@
-import { useRef } from 'react';
+import { TestimonialDto } from '@FullStackMap/from-a2b';
+import { Carousel } from '@mantine/carousel';
 import {
+  Badge,
+  Card,
+  Center,
   Container,
-  Title,
-  Text,
   Group,
+  Loader,
   Rating,
-  Center,
-  Card,
-  Badge,
+  Text,
+  Title,
 } from '@mantine/core';
-import { Carousel } from '@mantine/carousel';
+import { useQuery } from '@tanstack/react-query';
 import Autoplay from 'embla-carousel-autoplay';
+import { useEffect, useRef, useState } from 'react';
 import { FormFeedback } from '../../components/feedback/FormFeedback';
-
-const reviews = [
-  {
-    pseudo: 'John Doe',
-    date: '2023-02-01',
-    comment: 'La plateforme est très intuitive, je suis très satisfait !',
-    rating: 1,
-  },
-  {
-    pseudo: 'Jane Doe',
-    date: '2023-01-15',
-    comment:
-      'J’ai eu un problème avec ma réservation, mais le service client a été très réactif et m’a aidé à trouver une solution.',
-    rating: 1,
-  },
-  {
-    pseudo: 'Bob Smith',
-    date: '2022-12-20',
-    comment:
-      'Je suis un peu déçu par la qualité des services proposés, mais le rapport qualité-prix est correct.',
-    rating: 3,
-  },
-  {
-    pseudo: 'Alice Smith',
-    date: '2022-12-01',
-    comment:
-      'Je recommande vivement cette plateforme, j’ai passé un excellent séjour !',
-    rating: 5,
-  },
-  {
-    pseudo: 'Eva Brown',
-    date: '2022-11-15',
-    comment:
-      'Je suis très satisfaite de ma réservation, je n’hésiterai pas à revenir !',
-    rating: 5,
-  },
-];
-
-const averageRating =
-  reviews.reduce((acc, review) => acc + review.rating, 0) / reviews.length;
+import useNotify, { NotifyDto } from '../../hooks/useNotify';
+import { AnoTestimonialsController } from '../../services/BaseApi';
+import { AuthStore, useAuthStore } from '../../store/useAuthStore';
 
 const FeedbackPage = () => {
   const autoplay = useRef(Autoplay({ delay: 5000 }));
+  const { ErrorNotify } = useNotify();
+  const isLogged: () => boolean = useAuthStore((s: AuthStore) => s.isLogged);
+
+  const fetchTestimonials = async () => {
+    try {
+      const response = await AnoTestimonialsController.getAllTestimonialGET();
+      return response.data;
+    } catch (error) {
+      handleError();
+    }
+  };
+
+  const { data: reviews } = useQuery({
+    queryKey: ['Testimonials'],
+    queryFn: fetchTestimonials,
```

Comment:
Use Async func in this queryFn like this : 
```suggestion
    queryFn: async () => await machinController.TrucEndPoint(),
```